Source/Views/TextViews/SPTextStorage.swift

```diff
@@ -1,73 +0,0 @@
-//
-//  SPTextStorage.swift
-//  Sequel Ace
-//
-//  Created by Jakub Kaspar on 04.12.2020.
-//  Copyright © 2020 Sequel-Ace. All rights reserved.
-//
-
-import Foundation
-
-@objc final class SPTextStorage: NSTextStorage {
-
-	private var storage = NSTextStorage()
-
-	// MARK: - Required overrides for NSTextStorage
-
-	override var string: String {
-		return storage.string
-	}
-
-	override func attributes(at location: Int, effectiveRange range: NSRangePointer?) -> [NSAttributedString.Key : Any] {
-		return storage.attributes(at: location, effectiveRange: range)
-	}
-
-	override func replaceCharacters(in range: NSRange, with str: String) {
-		beginEditing()
-		storage.replaceCharacters(in: range, with: str)
-		edited(.editedCharacters, range: range, changeInLength: (str as NSString).length - range.length)
-		endEditing()
-	}
```

Comment:
Was this block the issue? Instead of reverting the whole double click thing, was there not just some part of this we could have reverted/altered? The double click changes were quite nice

Reply:
Swift is the issue, and subclassof NSTextStorage is the issue